package.json

```diff
@@ -102,11 +102,13 @@
     "libphonenumber-js": "^1.7.19",
     "moment": "^2.24.0",
     "node-sass": "^4.11.0",
+    "notifications-node-client": "^4.6.0",
     "nunjucks": "^3.2.0",
     "ramda": "^0.26.1",
     "redis": "^2.8.0",
     "request": "^2.88.0",
     "request-promise": "^4.2.2",
+    "uuid": "^3.3.2",
```

Comment:
Do we need both `uuidv4` and `uuid`?

Reply:
Didn't noticed we already had uuidv4, I'll remove uuid.